pmpsui/pmps.py

```diff
@@ -197,55 +195,55 @@ def setup_tabs(self):
     def setup_fastfaults(self):
         # Do not import Display subclasses at the top-level, this breaks PyDM
         # if using PyDMApplication + pydm as a launcher script
-        from fast_faults import FastFaults
+        from pmpsui.fast_faults import FastFaults
         tab = self.ui.tb_fast_faults
         ff_widget = FastFaults(macros=self.config)
         tab.layout().addWidget(ff_widget)
 
     def setup_preemptive_requests(self):
-        from preemptive_requests import PreemptiveRequests
+        from pmpsui.preemptive_requests import PreemptiveRequests
         tab = self.ui.tb_preemptive_requests
         pr_widget = PreemptiveRequests(macros=self.config)
         tab.layout().addWidget(pr_widget)
 
     def setup_arbiter_outputs(self):
-        from arbiter_outputs import ArbiterOutputs
+        from pmpsui.arbiter_outputs import ArbiterOutputs
         tab = self.ui.tb_arbiter_outputs
         ao_widget = ArbiterOutputs(macros=self.config)
         tab.layout().addWidget(ao_widget)
 
     def setup_ev_calculation(self):
-        from ev_calculation import EVCalculation
+        from pmpsui.ev_calculation import EVCalculation
```

Comment:
These internal imports have never really served a purpose in this application and have always confused me, I'd like to pull them out at some point as long as it doesn't cause any issues

Reply:
In an effort to make smaller PRs, I'd vote for deferring that to another PR